js/ui/appDisplay.js

```diff
@@ -548,6 +548,11 @@ const AllView = new Lang.Class({
         this._addedFolderId = null;
         IconGridLayout.layout.connect('folder-added',
                                       Lang.bind(this, function(iconGridLayout, id){
+                                          // Go to last page; ideally the grid should know in
+                                          // which page the change took place and show it automatically
+                                          // which would avoid us having to navigate there directly
+                                          this.goToPage(this._grid.nPages() - 1);
```

Comment:
The logic is obviously easier and in place for an append, rather than an inline addition that would depend on the position of the pointer, etc. I think we should keep this for this new release and work with the Design team is having an inline insertion is important.